docs/querying/multi-value-dimensions.md

```diff
@@ -25,7 +25,7 @@ title: "Multi-value dimensions"
 
 Apache Druid supports "multi-value" string dimensions. These are generated when an input field contains an
 array of values instead of a single value (e.g. JSON arrays, or a TSV field containing one or more `listDelimiter`
-characters).
+characters), by default Druid ingests the values in alphabetical order, see [Dimension Objects](../ingestion/index.md#dimension-objects) for configuration.
```

Comment:
This is a run on sentence. Please replace the comma with a period and split up the sentences.